src/dune_rules/lib_flags.mli

```diff
@@ -20,15 +20,21 @@ end
 
 (** Operations on list of libraries *)
 module L : sig
-  type nonrec t = Lib.t list
+  type nonrec t = (Lib.t * bool) list
```

Comment:
I am not convinced we want to change the type here (because some/most of the functions below do not make use of the boolean). Let's leave the type as before and change only the functions that absolute need to be changed. Later, we will see if we need to define a separate type for those, or move them to a separate module or...

Reply:
Yes; this change is leftover from before.